ot/torch/lp.py

```diff
@@ -79,3 +78,69 @@ def ot_solve(a, b, M, num_iter_max=100000, log=False):
         G = emd(a2, b2, M2, log=False, numItermax=num_iter_max)
 
         return torch.from_numpy(G).type_as(M)
+
+
+def emd1D_loss(u_values, v_values, u_weights=None, v_weights=None, p=1, require_sort=True):
+    r"""
+    Computes the 1 dimensional earth moving distance between two empirical distributions
+    ..math:
+        EMD &= \int_0^1 |cdf_u^{-1}(q)  cdf_v^{-1}(q)|^p dq
```

Comment:
ot_loss (this is actually the emd to the power of p)

Reply:
also add in the documentation that this works only for p>=1

Reply:
I need to even put an assert.

Reply:
that woudl be good indeed

Reply:
Done, I added a ref to Cuturi and Peyre for why it doesn't work if p < 1